### PR TITLE
We need CI for clang/windows.

### PR DESCRIPTION
**Pull request by lemire**

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -13,25 +13,34 @@ environment:
...
       SIMDJSON_PLATFORM: Win32
+      CMAKE_EXTRA_ARGS:
     - SIMDJSON_BUILD_STATIC: OFF
       SIMDJSON_ENABLE_THREADS: ON
       SIMDJSON_PLATFORM: x64
+      CMAKE_EXTRA_ARGS:
+    - SIMDJSON_BUILD_STATIC: OFF
+      SIMDJSON_ENABLE_THREADS: ON
+      SIMDJSON_PLATFORM: x64
+      CMAKE_EXTRA_ARGS: -T ClangCL
```
Comment (jkeiser) [inline .appveyor.yml:28]: I can't find this in the [cmake docs](https://cmake.org/cmake/help/v3.17/variable/CMAKE_GENERATOR_TOOLSET.html#variable:CMAKE_GENERATOR_TOOLSET), but if it works, it works :)

**Comment by jkeiser** [comment]
I ran into the multiplicity problem too, and ended up solving it with a single matrix in the checkperf PR so I had more control.

File: .appveyor.yml (hunk 1 above)
Comment (lemire) [reply .appveyor.yml:28]: Right. I tried many things. :-)

**Comment by lemire** [comment]
I don't understand why the fuzzers fail.

**Comment by jkeiser** [comment]
@lemire I'm guessing there was an interaction between two checkins when checkperf for Windows was merged. It [failed in master too.](https://github.com/simdjson/simdjson/commits/master)

**Comment by jkeiser** [comment]
Or maybe it wasn't an interaction and it just succeeded luckily in CI? The only intervening checkin seems to have nothing to do with the build.

**Comment by lemire** [comment]
This PR may never get merged. I would rather that we merge the one where I fix the problem with clang on Windows.

**Comment by lemire** [comment]
Closing in favor of https://github.com/simdjson/simdjson/pull/817

**Pull request closed without merging**